src/components/ButtonsGroup/ButtonsGroup.tsx

```diff
@@ -1,41 +1,52 @@
-import React, { Children, useMemo, useState } from 'react';
-import { FlatList, StyleSheet, View } from 'react-native';
-import { sizes } from '@magnetis/astro-tokens';
+import React, { useState } from 'react';
+import { FlatList, Pressable, PressableProps, StyleSheet, Text } from 'react-native';
+import { sizes, colors, radius } from '@magnetis/astro-tokens';
 
-import ButtonContent, { ButtonContentProps } from './ButtonContent';
+type Item = {
+  value: string;
+  onPressItem: () => void;
+};
 
-import type { Callback } from './types';
-
-export interface ButtonsGroupProps {
-  testID?: string;
-  children: React.FunctionComponentElement<ButtonContentProps>[];
+export interface ButtonsGroupProps extends PressableProps {
```

Comment:
Como o componente usa a nomenclatura no singular, sugiro ou mudar aqui ou mudar no componente.
```suggestion
export interface ButtonGroupProps extends PressableProps {
```

Reply:
Não foi ajustado 😅